backend/index.js

```diff
@@ -0,0 +1,44 @@
+const express    = require('express');
+const mongoose   = require('mongoose');
+const helmet     = require('helmet');
+const bodyParser = require('body-parser');
+const morgan     = require('morgan');
+const bluebird   = require('bluebird');
```

Comment:
Is there a reason to go with bluebird over built in promises?

Reply:
[Speed](https://softwareengineering.stackexchange.com/questions/278778/why-are-native-es6-promises-slower-and-more-memory-intensive-than-bluebird) and [more robust set of features and debugging](http://bluebirdjs.com/docs/features.html)

Reply:
Makes sense. Let's do it!